magenta/models/shared/melody_rnn_generate.py

```diff
@@ -125,13 +126,20 @@ def run_with_flags(melody_rnn_sequence_generator):
   if not os.path.exists(FLAGS.output_dir):
     os.makedirs(FLAGS.output_dir)
 
+
+# primer_sequence must be set to something, either primer_melody or primer_midi or midi #60 = 5C 
+# If set to midi #60 it will be converted to a random note in melody_rnn_sequence_generator.py
   primer_sequence = None
   if FLAGS.primer_melody:
     primer_melody = melodies_lib.MonophonicMelody()
     primer_melody.from_event_list(ast.literal_eval(FLAGS.primer_melody))
     primer_sequence = primer_melody.to_sequence()
   elif FLAGS.primer_midi:
     primer_sequence = midi_io.midi_file_to_sequence_proto(FLAGS.primer_midi)
+  else:  
+    primer_melody = melodies_lib.MonophonicMelody()
+    primer_melody.from_event_list('[60]')
```

Comment:
I thought you wanted to have it randomly sample the first note if no primer is given.